daml-assistant/daml-project-config/DA/Daml/Project/Consts.hs

```diff
@@ -43,6 +44,11 @@ import DA.Daml.Project.Types
 damlPathEnvVar :: String
 damlPathEnvVar = "DAML_HOME"
 
+-- | The DAML_CACHE environment variable determines the path of the daml
+-- assistant cache directory. This defaults to DAML_HOME.
```

Comment:
If we want a chance of this working automatically, I think we should default this to `XDG_CONFIG_HOME` or `XDG_CACHE_HOME`. Then an enterprise setup could customize `DAML_HOME` to ensure binaries are not user writable while `DAML_CACHE` will most likely not need customization.

The one thing I’m not entirely sure about is backwards compatibility but tbh, I think we can probably just ignore that given that we only write telemetry files there atm and writing those again is perfectly fine.

Reply:
agreed!